__tests__/app.test.js

```diff
@@ -136,6 +135,39 @@ describe('/api/articles/:id', () => {
     })
 })
 
+describe('/api/articles/:id/comments', () => {
+    describe('GET', () => {
+        test('status: 200 - responds with an comments object depending on ID of article.', () => {
+            return request(app)
+            .get('/api/articles/3/comments')
+            .expect(200)
+            .then((response) => {
+                    expect(response.body.comments).toMatchObject({
+                        comment_id: expect.any(Number),
+                        author: expect.any(String),
```

Comment:
How do you know you're getting back the correct comment object? 
This would match for all the comment objects, even if wrong.

You need to expect article_id: 3